### ISSUE TITLE
Manage product perks

### ISSUE THREAD

**Issue by Tirodem** [be-BOP-io-SA/be-BOP#288]
TBD

tl;dr : like Ulule / Kickstarter, as a pack of products instead of each product one by one during a challenge.

Idea :
- creation of an object "perk"
- includes a perk global description
- include products on a perk
- define a discount % for this perk, applyed on the sum of each product price
- develop a specific widget (inspired by the product widget) to display perk on CMS page
- when using a "Add to cart" CTA on a perk widget, it adds each product + a product "Perk discount" with a negative price
  - if one product is removed from the cart, the perk is not complete anymore, so the perk discount is removed
  - if you add manually each product composing a perk, the pek discount applies automatically
  - in fact you can consider the perk as a group discount, a group of product, or an advanced promotional ruleset on cart that can be used outside crowdfunding behavior

Proto design for widget :
![image](https://github.com/B2Bitcoin/B2BitcoinBootik/assets/50206014/f0033bfa-eea2-44f6-a6c2-7dd03b3f0d73)

I need to dig in order to evaluate the impact of perks on challenge gauges (remove perk discount amount from the money amount challenge current value ?)

**Comment by Tirodem** [comment]
Will be best covered by https://github.com/be-BOP-io-SA/be-BOP/pull/2246
